templates/x86/qemu/mods.config

```diff
@@ -73,6 +74,8 @@ configuration conf {
 	@Runlevel(1) include embox.test.stdlib.qsort_test
 	@Runlevel(1) include embox.test.posix.environ_test
 	@Runlevel(1) include embox.test.posix.timerfd_test
+	//include embox.test.posix.forktest_fork_static
+	//@Runlevel(1) include embox.test.posix.fork_test
```

Comment:
is it work If it's uncommented?
may be add also **@Runlevel(1) include embox.test.posix.vfork_test**